boards/arm/sam_e70_xplained/sam_e70_xplained.dts

```diff
@@ -65,6 +65,10 @@
 	status = "ok";
 };
 
+&spi0 {
+	status = "ok";
```

Comment:
How are the chipselects wired on the board?  Using gpio? if so we should describe them here.

Reply:
There are nothing on spi bus, so I only enable spi0 and doesn't add any other nodes.

Besides: How to detect using gpio cs or using hardware cs control in a spi driver?